.github/workflows/c3-e2e.yml

```diff
@@ -19,12 +19,11 @@ jobs:
         continue-on-error: true
         run: |
           npx turbo-ignore create-cloudflare --task="test:e2e" --fallback=$(git merge-base origin/main HEAD)
-
-  e2e:
+  e2e-vp:
     # Note: please keep this job in sync with the e2e-only-dependabot-bumped-framework one
     #       in .github/workflows/c3-e2e-dependabot.yml
     needs: ["turbo-ignore"]
-    if: ${{ needs.turbo-ignore.outputs.skip == 'failure' }}
+    if: needs.turbo-ignore.outputs.skip == 'failure' && github.head_ref == 'changeset-release/main'
```

Comment:
should we check the org is cloudflare too?

Reply:
I don't think we need to? The C3 tests shouldn't depend on any actions secrets, I don't think (although I may be mistaken there?)